src/subject_level/getBoldFilenameForFFX.m

```diff
@@ -13,9 +13,6 @@
   runs = getInfo(BIDS, subID, opt, 'Runs', sessions{iSes});
 
   prefix = getPrefix('FFX', opt, funcFWHM);
```

Comment:
I did not understand well what getPrefix would do exactly at the current form. BUT I guess we will still have different folders for MNI-spaced FFX and individuals-spaced ;) FFX folders. Right?

Reply:
Yes and getPrefix is not related to that. It is mostly to help figure out what the prefix is when looking for a functional file that needs to be normalized.

The name of the directory is handled by `getFFXdir` and it does include the name of the `space` in the folder name.